app/src/main/AndroidManifest.xml

```diff
@@ -55,6 +55,9 @@
         <service android:name=".ui.widgets.ChargeWidgetService"
                  android:permission="android.permission.BIND_REMOTEVIEWS" />
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="8dbc177cd5391a21947da95f2aa9208f33cd45f4" />
```

Comment:
Put this key to https://github.com/openMF/self-service-app/blob/development/app/src/debug/res/values/google_maps_api.xml and Make the file name `google_maps_api.xml` -> `api_keys.xml`, In both debug and release.